docs/reference/reranking/index.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[re-ranking-overview]]
+= Re-ranking
+
+Many search systems are built on two-stage retrieval pipelines.
+
+The first stage uses cheap, fast algorithms to find a subset of possible matches.
+
+The second stage uses a more powerful model, often machine learning-based, to reorder the documents.
+This second step is called re-ranking.
+This approach balances computational costs, because the resource-intensive model is only applied to a smaller set of pre-filtered results.
+
+{es} supports various ranking and re-ranking techniques to optimize search relevance and performance.
+
+[float]
+[[re-ranking-two-stage-pipeline]]
+== Two-stage retrieval pipelines
+
+Learn about retrieval pipelines and how re-ranking can be applied to your existing search experience.
```

Comment:
Starting this sentence with `Learn about` this deep into the document feels a bit awkward. Do we need it?

Reply:
Yeah definitely a placeholder kinda glue sentence...I'll ponder alternative

Reply:
let's remove it entirely